src/SprykerEco/Zed/Computop/Business/Order/Mapper/PostPlace/PayPalExpressMapper.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Order\Mapper\PostPlace;
+
+use Generated\Shared\Transfer\PaymentTransfer;
+use SprykerEco\Shared\Computop\ComputopConfig;
+use SprykerEco\Zed\Computop\Business\Order\Mapper\MapperInterface;
+
+class PayPalExpressMapper implements MapperInterface
+{
+    /**
+     * @return string
+     */
+    public function getMethodName(): string
+    {
+        return ComputopConfig::PAYMENT_METHOD_PAY_PAL_EXPRESS;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PaymentTransfer $paymentTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer
+     */
+    public function getComputopTransfer(PaymentTransfer $paymentTransfer)
```

Comment:
Eather change this return docblock to TranferInterface or change the docblock of next method to be more specific